README.md

```diff
@@ -1,3 +1,3 @@
 # List of books to read
 
-1. So good they can't ignore you - Cal Newport
+1. [So good they can't ignore you](https://www.amazon.in/Good-They-Cant-Ignore-You) - Cal Newport
```

Comment:
Can you add few more books along with references ?

Reply:
Yes, I will be working on books with references to its audio and video links too.